man/rshim.8

```diff
@@ -36,10 +36,13 @@ Display the output
 cat /dev/rshim0/misc
     DISPLAY_LEVEL   0 (0:basic, 1:advanced, 2:log)
     BOOT_MODE       1 (0:rshim, 1:emmc, 2:emmc-boot-swap)
-    BOOT_TIMEOUT    100 (seconds)
+    BOOT_TIMEOUT    150 (seconds)
+    DROP_MODE       1 (0:normal, 1:drop)
```

Comment:
Maybe display "DROP_MODE   0" as default setting?

Reply:
Resolved in 4dd1482